### ISSUE TITLE
yumrepo not being installed with pip install of ansible 2.0.0.2

### ISSUE THREAD

**Issue by nralbers** [ansible/ansible-modules-extras#1519]
Ansible gives a "no action detected in task" when using the action yumrepo. Checking in site-packages/ansible there is no yumrepo.py file present on the server.

Ansible was installed using pip install ansible.

```
- name: configure logstash yum repo
  yumrepo:
    name: logstash-2.1
    description: Logstash 2.1 repo
    baseurl: http://packages.elastic.co/logstash/2.1/centos
    gpgcheck: yes
    gpgkey: https://packages.elastic.co/GPG-KEY-elasticsearch
```

**Comment by gtirloni** [comment]
yumrepo will be available only in 2.1. See #1016

**Comment by abadger** [comment]
Yep, it's a new module and was merged after we had closed 2.0.x for new features.  Most modules for 2.1 are compatible with 2.0 so if you need it you should be able to copy it from the repository locally and make use of it.
